### ISSUE TITLE
W kerning pairs

### ISSUE THREAD

**Issue by August-Alm** [dbenjaminmiller/xetbook#1]
![20190722_100722](https://user-images.githubusercontent.com/24792368/61616410-9a7d7400-ac68-11e9-9950-df9501cf5c45.jpg)
The kerning between capital W and lower case letters like o, e and a needs to be decreased. See attached screenshot of "Would We Want."

**Comment by August-Alm** [comment]
![Screenshot_20190722-101531_FontViewer](https://user-images.githubusercontent.com/24792368/61616853-d7963600-ac69-11e9-9f36-057e3683bd4b.jpg)
Here are some more kerning pairs between caps and lower case letters that need to be fixed.

**Comment by August-Alm** [comment]
I looked at the font again now and realized it has no kerning at all. I'm not very savy with Fontforge and other such software but would it be possible to simply copy the kerning data from a similar Bembo-like font? As a first approximation of perfection. The glyphs are beautiful but without kerning the font is not really useable.

**Comment by dbenjaminmiller** [comment]
You are right. I have not gotten around to doing it yet. My main concern is that copying the kerning from another font would run afoul of licensing restrictions ­— and I don't want to change the terms I offer the fonts under, nor make enemies with the commercial foundries. If you want to, you can kern it yourself. Otherwise, I will get around to it at some point…
